tests/unit/ops/cuda/test_cuda_backward.py

```diff
@@ -286,9 +287,8 @@ def test_backward(self,
                       is_preln,
                       use_fp16,
                       atol):
-        # Only run fp16 test cases on devices with 7+ capability.
-        major, _ = torch.cuda.get_device_capability()
-        if major < 7 and (use_fp16 is True or is_preln is False):
+        # Only run fp16 test cases on devices with FP16 capability.
```

Comment:
Is it okay that `is_preln` test is dropped here?

Reply:
1. is_preln should not be dropped, will fix this place, thanks.
2. The list of CUDA compute capability could be found here https://docs.nvidia.com/cuda/cuda-c-programming-guide/index.html#compute-capabilities From the table on the page, capability 7 start from v100, and P100 is 6.  I guess 7 is the generation that FP16 is preferred over FP32.